app/controllers/amp_articles_controller.rb

```diff
@@ -1,4 +1,7 @@
 class AmpArticlesController < ActionController::Base
+
```

Comment:
empty line was intentional left, makes it easier to read and debug, but that's just my opinion :)